strongbox-commons/src/main/java/org/carlspring/strongbox/artifact/coordinates/AbstractArtifactCoordinates.java

```diff
@@ -33,13 +31,15 @@ public AbstractArtifactCoordinates(Map<String, String> coordinates)
         this.path = toPath();
     }
     
-    protected final void defineCoordinates(String... coordinates)
+    protected final void resetCoordinates(String... coordinates)
```

Comment:
Why is this renamed?

Reply:
my thoughts was that this method reset all coordinates to `null`, so it's not **set** or **define** but more like  **reset**. 
I can return it back if you don't like this name.

Reply:
(I was just trying to understand what's goign on :) ).